### ISSUE TITLE
Light sensor on LD2410

### ISSUE THREAD

**Issue by msherga** [kippesikgithub/esp_motion_mmwave#5]
Per this page: https://esphome.io/components/sensor/ld2410.html

there is a light sensor (LDR) that can be referenced. Any idea how to include this in the config? The docs make it seem as though you just add it to the yaml, but I assume the config would need updating first. Have you ever implemented it?

Thanks for sharing your work!

**Comment by kippesikgithub** [comment]
nope, never used it, having Lux sensors all through the house. But if I read correct, when you put the board in engineering mode, you can edit the lux sensitivity. Since i'm not using the native LD2410 esphome packages, you should convert the whole config to the native component first.

**Comment by kippesikgithub** [comment]
I updated all the code and instructions to now use the native ld2410 component in esphome.
All the output entities are also added, including an prefix for all the entities. Take a look, and start using these for now. The entites from the old config, willbe named the same in this new one.
Also including the light sensor, if you still have this question
